stdlib/src/builtin/int.mojo

```diff
@@ -630,17 +622,9 @@ struct Int(
             rhs: The value to divide on.
 
         Returns:
-            The quotient and remainder as a `Tuple(self // rhs, self % rhs)`.
+            The quotient and remainder as a tuple `(self // rhs, self % rhs)`.
         """
-        if rhs == 0:
-            return 0, 0
-        var div: Int = self._positive_div(rhs)
-        if rhs > 0 & self > 0:
-            return div, self._positive_rem(rhs)
-        var mod = self - div * rhs
-        if ((rhs < 0) ^ (self < 0)) & mod:
-            return div - 1, mod + rhs
-        return div, mod
+        return self // rhs, self % rhs
```

Comment:
**Question** By recomputing things, won't this regress performance a bit?

Reply:
Both `//` and `%` are marked as `always_inline`; I just assume that the compiler can see through the and do CSE well.

Reply:
As a ref point, `UInt` also does this.

Reply:
I checked and they generates identical assembly.